packages/blade/src/components/Dropdown/DropdownOverlay.web.tsx

```diff
@@ -38,7 +38,7 @@ const _DropdownOverlay = ({
   const { theme } = useTheme();
   const bottomSheetAndDropdownGlue = useBottomSheetAndDropdownGlue();
 
-  const isMenu = dropdownTriggerer !== 'SelectInput';
+  const isMenu = dropdownTriggerer !== 'SelectInput' && dropdownTriggerer !== 'AutoComplete';
```

Comment:
Wondering if we should store dropdownTrigger as an enum. Its getting big now

Reply:
we can add if more triggers come up. I don't think so apart from Select and AutoComplete there will be any other input trigger anytime soon 🤔

Reply:
I moved it to constants